### DISCUSSION TITLE
Quitting Rustdesk on Windows causes service removal leads to "lost control disaster"

### DISCUSSION THREAD

**Discussion by Kylejustknows** [rustdesk/rustdesk#8095] [General]
### Bug Description

It is so easy to exit Ruskdesk on a Windows server.
But doing that the Ruskdesk services is automatically removed.
There is no way to connect back to that machine again, even remote restart the VPS can not gain back control.

Has to go through a painful backend screen control to install rustdesk service again.

### How to Reproduce

Exit Rustdesk on Windows using the Tray Icon. 
It will somehow Uninstall Rustdesk completely. 

### Expected Behavior

The

- End user space task (disconnect temporarily), and 
- Uninstall Rustdesk service (disconnect permanently)

Should be two different tasks.
The uninstalling should give much clear warning.

The "Rustdesk service" existing gives user a false impression that the service should be there even exit the user app.

### Operating system(s) on local side and remote side

Windows 10

### RustDesk Version(s) on local side and remote side

1.2.4

### Screenshots

![image](https://github.com/rustdesk/rustdesk/assets/14979096/5e5b4fb5-a185-4710-b94a-fab5ba704f91)

**Comment by MikeT-KBU** [comment]
Why is there no reaction? We have Pro-Version now and there is exact same Problem. If someone klick on exit we can't get on Server ever again. This is a really big problem for us.

**Comment by radmedics01** [comment]
We're testing RustDesk - but this is a showstopper so there is no way to migrate from TeamViewer. Why isn't this classified as a bug?

**Comment by Coldblackice** [comment]
This is such a bizarre function, and even more bizarre that there isn't a setting to stop it. 

Why in the world is it assumed that manually exiting RustDesk == not wanting RustDesk (service) to autostart?
